test/OpenTelemetry.AutoInstrumentation.Tests/OtelLoggingTests.cs

```diff
@@ -0,0 +1,125 @@
+// <copyright file="OtelLoggingTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using FluentAssertions;
+using OpenTelemetry.AutoInstrumentation.Logging;
+using Xunit;
+
+namespace OpenTelemetry.AutoInstrumentation.Tests;
+
+public class OtelLoggingTests
+{
+    [Fact]
+    public void WhenNoFileSizeIsConfigured_Then_DefaultIsUsed()
```

Comment:
Consider commonly used IDisposable tests for env. vars.

Reply:
changed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2277/commits/c912cfecbdb689d1c5cb6491c633c10cf3ed9fc4